sovrin_common/persistence/identity_graph.py

```diff
@@ -626,6 +626,12 @@ def addSchemaTxnToGraph(self, txn):
         origin = txn.get(f.IDENTIFIER.nm)
         txnId = txn[TXN_ID]
         data = txn.get(DATA)
+        if isinstance(data, str):
```

Comment:
I see almost duplicate code in this method and the other one where you introduced changes. Can you please reuse the code.